ui/app/components/auth-form.js

```diff
@@ -206,12 +211,15 @@ export default Component.extend(DEFAULTS, {
   showLoading: or('isLoading', 'authenticate.isRunning', 'fetchMethods.isRunning', 'unwrapToken.isRunning'),
 
   authenticate: task(
-    waitFor(function* (backendType, data) {
-      let clusterId = this.cluster.id;
+    waitFor(function* (backend, data) {
```

Comment:
I do like that the previous name was clear that this is the backend type, not the backend name. Could we add a comment or keep the old name here?

Reply:
Sure I can change it back. The auth service `authenticate` method has it named `backend` so I thought I would align it that way.